### ISSUE TITLE
Convert worklog.adoc into formal, trackable issues.

### ISSUE THREAD

**Issue by jaredmorgs** [opendevise/docbookrx#22]
Atomising the Worklog into discreet tasks may just allow some contributors to have a go at resolving them. And it will look great when you do a release: all those bugs!!! ;)

**Comment by mojavelinux** [comment]
I agree. That's definitely the plan. I should probably call the file BANKRUPT.adoc because the only reason it exists is because I haven't had a chance to file the issues (mostly from issues that I jotted down before I had a repository setup).

**Comment by jaredmorgs** [comment]
I'll do it for you. We've seen what it did for the docs project, and this
is a critical tool from an adoption perspective IMHO.

On Thu, 3 Mar 2016, 04:39 Dan Allen <notifications@github.com> wrote:

> I agree. That's definitely the plan. I should probably call the file
> BANKRUPT.adoc because the only reason it exists is because I haven't had a
> chance to file the issues (mostly from issues that I jotted down before I
> had a repository setup).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opendevise/docbookrx/issues/22#issuecomment-191365802>
> .
>
-- 

Sent from Mobile.

**Comment by mojavelinux** [comment]
That would be great. First, though, let me scan through it and mark items that should be transferred. Some of them I haven't committed to doing yet, and may have since decided not to do, so I want to downgrade those. I'll let you know when I push the update.

**Comment by jaredmorgs** [comment]
All good. Ping me once ready.

On Thu, 3 Mar 2016, 07:37 Dan Allen <notifications@github.com> wrote:

> That would be great. First, though, let me scan through it and mark items
> that should be transferred. Some of them I haven't committed to doing yet,
> and may have since decided not to do, so I want to downgrade those. I'll
> let you know when I push the update.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opendevise/docbookrx/issues/22#issuecomment-191443835>
> .
>
-- 

Sent from Mobile.

**Comment by mojavelinux** [comment]
I pushed an update that marks the first batch of candidate issues with a trailing asterisk.

Some of the other items I'm not even sure what I meant exactly, or they aren't really a priority, so I didn't mark those.

See https://github.com/opendevise/docbookrx/blob/master/WORKLOG.adoc#todo

**Comment by jaredmorgs** [comment]
Carpe Diem. The bugs are now created @mojavelinux 👍

**Comment by mojavelinux** [comment]
:+1:
